wasm/nodejs/refactoring/utxo-split.js

```diff
@@ -23,72 +24,79 @@ let {
 } = kaspa;
 kaspa.init_console_panic_hook();
 
-(async ()=>{
-    
-    let URL = "ws://127.0.0.1:17110";
-    let rpc = new RpcClient(Encoding.Borsh,URL);
-    
+(async () => {
+    const {
+        encoding,
+        address,
+    } = parseArgs();
+
+    const URL = "ws://127.0.0.1:17110";
+    const rpc = new RpcClient(encoding, URL);
+
     console.log(`# connecting to ${URL}`)
     await rpc.connect();
-    
-    let info = await rpc.getInfo();
+    await guardRpcIsSynced(rpc);
+
+    const info = await rpc.getInfo();
     console.log("info", info);
-    
-    let addresses = [
-        new Address("kaspa:qq5dawejjdzp22jsgtn2mdr3lg45j7pq0yaq8he8t8269lvg87cuwl7ze7djh"),
-        new Address("kaspa:qzewpzt0rx6jmvy0eea82lpnf0t7f7frmqavqcaawmt4wk70puazcp8zljgx5"),
+
+    const addr1 = new Address(address ?? "kaspa:qq5dawejjdzp22jsgtn2mdr3lg45j7pq0yaq8he8t8269lvg87cuwl7ze7djh");
```

Comment:
replace `new Address()` with `"kaspa:xxxx"` (everywhere)

(and then we need to test to make sure the functions are accepting strings).